examples/README.md

```diff
@@ -0,0 +1,48 @@
+# Merlin Models Example Notebooks
+
+We have created a collection of Jupyter notebooks based on different datasets. These example notebooks demonstrate how to use Merlin Models with TensorFlow.
+
+## Available Example Notebooks
+
+1. **[Getting Started](01-getting-started.ipynb)**: It provides a getting started with Merlin Models. We will train [Facebook's DLRM](https://arxiv.org/pdf/1906.00091.pdf) architecture with only 3 commands.
+
+2. **[Merlin Models and NVTabular: Applying To your own dataset](02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb)**: We will take a look on the `schema` to connect ETL and training step. It will help to apply Merlin Models to your own dataset structures.
+
+3. **[Exploring different Models](Exploring-different-models.ipynb)**: We will explore different ranking model architectures, such as [Neural Collaborative Filtering (NCF)](https://arxiv.org/pdf/1708.05031.pdf), MLP, [DRLM](https://arxiv.org/abs/1906.00091) and [Deep & Cross Network (DCN)](https://arxiv.org/pdf/1708.05123.pdf)
```

Comment:
3. **[Iterating over Deep Learning Models using Merlin Models](...)**: Explore several ranking model architectures such as...

Please add a serial comma after the link for DLRM.